### DISCUSSION TITLE
Site allocations

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#42] [Planning consideration]
# Site allocations

The Site Allocations are designations made by Local planning authorities. Local planning authorities follow a process to determine whether sites would be earmarked for housing and other uses, such as retail and employment uses, and can include site-specific proposals

## What we're looking for

We'd like to know:

* what does `Site Allocations` mean to you?
* what sort of data is needed for the Site Allocations process?
* what data should be the output of this process?
* does any Site Allocation data already exist? If so, where and how is it used?
* what would you use Site Allocation for?
* what should be included in Site Allocation data?
* anything else you think we should know

**Comment by drewhardy77** [comment]
To me, site allocations are a formal component of a Local Plan, detailing the areas which the planning authority has deemed suitable for various types of development during the plan period. Sites are allocated after undergoing various stages of assessment, including availability, achievability and suitability assessments.

Site allocations give increased certainty to developers, in that they can understand what kind of development is likely to be approved at a given site, and increase the likelihood of their application being approved compared to speculative applications in non-allocated sites. Additionally, the allocations data allows a planning authority to manage development strategically.

An example of existing site allocations shared as open data can be found on [Nottingham City Council's data sharing platform](https://geoportal-nottmcitycouncil.opendata.arcgis.com/datasets/ab6121be95df411182b56790089c60b7_41/explore) 

Components I would expect to see within site allocation data:

- Polygon geometry
- Unique persistent reference
- Current use
- Proposed use e.g. broad category such as residential, mixed use, employment
- Ownership e.g. broad category such as council, MoD, private, commercial
- Maximum number of dwellings
- Minimum number of dwellings
- Maximum office space
- Minimum office space
- Maximum retail space
- Minimum retail space
- Maximum industrial/warehouse space
- Minimum industrial/warehouse space
- Date record created
- Date record ends (following development)
- Ward name (this should be derived from spatial data intersection not manually added) 
- Ward code (this should be derived from spatial data intersection not manually added) 
- Known constraints
- Brownfield register reference
- Planning permission reference(s)
- Link to assessment documentation
- Relevant local plan policy reference

Not all fields will be relevant for all allocations.

**Reply by drewhardy77** [reply]
[Mural link](https://app.mural.co/t/mhclg2837/m/mhclg2837/1748008910372/edfc4b41452170321c6fd398c2f06c36207cfe6b?sender=u1b53f3ea33df906141455566) to flow diagram showing a simplified representation of the allocation process

**Reply by drewhardy77** [reply]
Would net numbers as well as gross for dwellings/space be a useful addition?

**Reply by AngusMcNeillPeel** [reply]
Net additional dwellings / floorspace for employment, retail, leisure etc. would be very useful because for the purposes of meeting local needs, net is more important than gross. Some sites may result in the loss of existing uses via demolition, repurposing or change of use.

**Reply by AngusMcNeillPeel** [reply]
I agree with the description data required above, I would also suggest that one would expect to see address, postcode, area in hectares, timescale for development, known viability issues

**Reply by drewhardy77** [reply]
With regard to 'viability', are you referring to economic viability or the presence of known constraints/suitability issues? I only ask as Claire was keen to keep 'viability' as a term referring to economic viability rather than wider issues

**Reply by AngusMcNeillPeel** [reply]
I mean economic viability. I can think of an example of where an issue would be economic viability - say 'this site requires significant land remediation and demolition to come forward' or 'this site requires a new link road to be constructed to come forward'. These generate large additional costs that may render development unviable. But they are not constraints/suitability issues - just enabling works that would generate a major cost implication. Viability is a requirement of **PPG Housing and economic land availability **para 20 What factors should be considered when assessing achievability including whether the development of the site is viable?**** Viability would fall under the 'achievability' test rather than suitability or availability.

This states that: A site is considered achievable for development where there is a reasonable prospect that the particular type of development will be developed on the site at a particular point in time. This is essentially a judgement about the economic [viability of a site](https://www.gov.uk/guidance/viability), and the capacity of the developer to complete and let or sell the development over a certain period.

**Reply by drewhardy77** [reply]
`Developable area` would be a useful attribute, to account for any parts of the site which are constrained. Total area can be derived from the site polygon so is not required as an attribute

**Reply by AngusMcNeillPeel** [reply]
Agreed

**Comment by AngusMcNeillPeel** [comment]
Hi just adding my thoughts here.

**What does Site Allocations mean to you?** Based on the PPG, a Site Allocation means that the site is allocated by a plan-making body in a development plan document (such as the London Plan (but potentially other future strategic plans but see NPPF footnote 16), Local Plan, Area Action Plan, Neighbourhood Plan) for a specific type of development or use, such as housing, employment and leisure. Allocated sites provide additional certainty and guidelines for planning decisions, help to diversify use of land and promote development. They are a requirement of NPPF para 23 'This should include planning for and allocating sufficient sites to deliver the strategic priorities of the area'. Allocations are also a critical tool for planning for a sufficient number of homes - para 69. Neighbourhood plans are also encouraged to allocate to meet local needs - para 74. More broadly, site allocations are a critical top line aspect of demonstrating that a plan has a deliverable pipeline of housing, employment and town centre sites that will support the strategic housing and economic policies of the plan over the course of the plan period. Where an authority cannot demonstrate it will meet its LHN through completions and commitments, it will need site allocations to demonstrate a sufficiently robust pipeline, particularly within the 5 year timeframe.

**What sort of data is needed for the Site Allocations process?** 

Site allocations are the result of a complex process that stems from - calls for sites - invitations to landowners and site promoters to put forward sites for development, and site assessment - often called a SHLAA or HELAA, which provides a robust assessment of suitable, available and achievable sites in accordance with the guidance set out by PPG Housing and economic land availability assessment. Once a plan-making body has selected its sites for allocations - these are consulted upon. Revisions to allocations may take place as part of revisions to a plan during consultation stages and examination in public. They can form a particularly complex part of discussions between the plan-making body, statutory consultees and other third parties over infrastructure requirements and site specific design requirements.

This means that technically speaking, anyone engaging in the housing and economic land availability assessment process needs access to various geographical data to undertake the full assessment - including statutory environmental constraints (e.g. SSSI, Ancient Woodland), statutory historic designations (e.g. Scheduled Monuments, Listed Buildings), topography, flood risk zones etc. 

For those trying to engage with the site allocation process, ideally they would have ready access to the plan-making bodies list of all available sites, their site assessment ratings and descriptions, and their site allocations. These are currently required to be published as part of the plan evidence - which typically gets published at the point of submission of the plan for examination in public and in PDF form. It is rare that the information would be available mapped digitally unless the plan-making body engages in a digital consultation exercise.

**What would the output be?** The output should be a publicly accessible record of site allocations.

**Does any Site Allocation data already exist? If so, where and how is it used?** I am less sure on this but it does exist largely in traditional PDF format - in the SHLAAs and HELAAs that LPAs publish, and in the SOAs that Neighbourhood Plan groups publish. And of course, in the plan documents and policy maps themselves.

**What would you use Site Allocation for?** For a variety of purposes as they are one of the most critical tools of the English plan-making system - chiefly to provide additional certainty over the location, scale and quantum of different types of development in the plan area over the plan period, delivering development in a plan-led way rather than through speculative development, providing additional certainty to applicants and decisionmakers over where development is appropriate. 

This is a fairly quick summary given how important site allocations are to the plan-making system so I may come back to this with some more thoughts!

**Reply by cjsharpe** [reply]
Hi - very interested in the potential of this.

> what would you use Site Allocation for?

If site allocation data was available in a consistent format through an API then it would enable the development of apps for several purposes, for example:

1. It could significantly speed up the process for housing developers looking for sites, as new tools could automatically search for allocated sites within a given size range, and filter them by other criteria relevant to the company (demographics, proximity to services, transport etc).
2. It could be used for market entry analysis by retailers or hotel operators, for example a supermarket looking to identify high growth areas where they might open a new store.
3. It could be used to monitor the distribution of potential housing growth at a regional scale across local planning authority boundaries, which could inform longer term demand modelling for strategic transport, health or educational infrastructure.

> what should be included in Site Allocation data?

The components described above by @drewhardy77 look like a great starting point (based on the Nottingham data) but I'd make a couple of suggestions:
1. The way that land uses and quantities are described varies across planning authorities - for example some might use detailed planning use classes (C3, A1 etc), some might specify other uses ('Cinema', 'Community', 'Health' etc), and sometimes there may not be a specific min or max, but just an indicative quantity. The data model would need to flexible enough to cope with a range of descriptions.
2. Ownership could be fragmented across the site allocation, so another category like 'Multiple Ownership' could be needed.
3. Ideally there should be links to any design policies relating to the site, such as design codes or other guidance that restrict scale, height, density, architectural character or layout. For example, the [Camden Site Allocations](https://www.camden.gov.uk/documents/20142/4820180/Site+Allocations+Plan+2013.pdf) provide very detailed guidance on each site.

**Comment by drewhardy77** [comment]
Requests from ALIGN - to be elaborated further:

**Residential**

housing site allocation polygons and underlying data on:
- number of homes
- densities
- start year
- build-out rates from existing and new local plans

**Employment**

as above but for employment sites to also include:
- number of jobs and when by, 
- potentially commercial floorspace as well

**Reply by AngusMcNeillPeel** [reply]
Interesting Drew - where did this stem from?

**Reply by drewhardy77** [reply]
These are requests from the National Infrastructure and Service Transformation Authority group. Their interest is in understanding exactly what infrastructure will be required to support proposed development, and they require this for existing allocations as well as allocations in new local plans

**Comment by AdeleMarr** [comment]
Site allocation data is also used as baseline data for the development of spatial development strategies (SDS). Also for monitoring what has been allocated, when, for what, has it been delivered.

**Comment by drewhardy77** [comment]
**Update from NPPF 2026**

A broad outline of a specification for allocations is noted in footnote 8 for [Section PM2 :Local Plans](https://assets.publishing.service.gov.uk/media/6a8334c03bd75b81e2329ac4/National_Planning_Policy_Framework.pdf#page=10)

```
Allocations should identify any site-specific expectations and requirements, including for on-and off-site 
infrastructure, and should include: a site reference, name and area; a short description of existing and allocated 
use(s); an indicative or specific site capacity for the development proposed; and the projected delivery 
timeframe.  
```
